### ISSUE TITLE
Unfocusing video on YouTube / no frame hints?

### ISSUE THREAD

**Issue by The-Compiler** [The-Compiler/qutebrowser#1014]
When a video has keyboard focus on YouTube (i.e. after clicking it), using `hjkl` will change volume/position instead of scrolling because of how it sends fake keypresses.

That's probably something I can't do much about, but there should be a way for the user to focus the page to scroll without using the mouse. Usually, each frame would get a hint at the top left, but I don't see such a thing on YouTube.

**Comment by NoSuck** [comment]
I agree.  It would be very nice to have the ability to change the element with focus.  YouTube is not the only [affected page.](https://www.bing.com/images/search?q=qute)

**Comment by lahwaacz** [comment]
This might be the same as [#218 (comment)](https://github.com/The-Compiler/qutebrowser/issues/218#issuecomment-176443564), except the video obviously handles the fake keypresses and input fields don't.

**Comment by NoSuck** [comment]
Bing has updated its image search page to no longer use frames, so the link I previously used as an example is no longer valid.  Yay!

**Comment by amacfie** [comment]
workaround: focus another element with <tab> (sometimes works better in insert mode)

**Comment by VuiMuich** [comment]
Another workaround (at least on youtube) is to `10G` 10%-scroll, then open the "more" link of the video description.
